solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/issue_pass.rs

```diff
@@ -1,12 +1,47 @@
 use crate::constants::PASS_SEED;
-use crate::state::{Gatekeeper, GatekeeperKeyFlags, GatekeeperNetwork, Pass, PassState};
+use crate::state::{Gatekeeper, GatekeeperNetwork, NetworkFees, Pass, PassState};
+use crate::util::{
+    calculate_network_and_gatekeeper_fee, create_and_invoke_transfer, get_gatekeeper_fees,
+    get_network_fees,
+};
 use anchor_lang::prelude::*;
 use anchor_lang::Key;
+use anchor_spl::token::{Mint, Token, TokenAccount};
 
+#[inline(never)]
```

Comment:
Added this and the Box<>'s in IssuePass cause of issues with memory

Reply:
What happens if this is removed? The Boxing should be enough (I think?)

Reply:
You are are right, we can leave out the `#[inline]`.

Reply:
I think I also like to understand the `Box<>` / `Heap` requirement.